impl_core/src/main/java/io/opencensus/implcore/internal/EventQueue.java

```diff
@@ -31,4 +31,12 @@
      */
     void process();
   }
+
+  /** No-op implementation of the {@link EventQueue.Entry}. */
+  enum NoopEntry implements Entry {
```

Comment:
Why do we need a no-op Entry?

Reply:
To fix the nullness warning. The entry is never going to be null.